web/src/engine/websites/MangaSefiri_e2e.ts

```diff
@@ -0,0 +1,25 @@
+﻿import { TestFixture } from '../../../test/WebsitesFixture';
+
+const config = {
+    plugin: {
+        id: 'adonisfansub',
```

Comment:
ID and Title did not match, test will fail...

Reply:
ooops. Was done during the time testing wasnt working. I made sure it worked and forgot to fix the test.